lnrpc/routerrpc/router.proto

```diff
@@ -22,6 +22,11 @@ service Router {
     */
     rpc TrackPaymentV2 (TrackPaymentRequest) returns (stream lnrpc.Payment);
 
+    /*
+    TrackPayments returns an update stream for every payment.
```

Comment:
nit: every payment that is not in a terminal state?

Reply:
Updated